cms/templates/js/move-xblock-list.underscore

```diff
@@ -0,0 +1,38 @@
+<div class="xblock-items-category">
+    <% // Translators: The complete message will be something like `Showing Subsections/Units/Components` etc. %>
+    <span class="sr"><%- gettext('Showing ') %></span>
```

Comment:
You shouldn't include partial sentences in translations because in some languages the sentence order will be reversed.